test/units/TEST-17-UDEV.verify.sh

```diff
@@ -293,8 +293,9 @@ test_syntax_error 'OWNER!="b"' 'Invalid operator for OWNER.'
 test_syntax_error 'OWNER+="0"' "OWNER key takes '=' or ':=' operator, assuming '='."
 # numeric system UID is valid even if it does not exist
 SYS_UID_MAX=999
-if [[ -e /etc/login.defs ]]; then
```

Comment:
to avoid introducing regressions it would be better to keep the login.defs as a fallback - ie, if userdbctl is not available, check if login.defs is